presto-analyzer/src/main/java/com/facebook/presto/sql/analyzer/AnalyzerClient.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.facebook.presto.sql.analyzer;
+
+import com.facebook.presto.spi.PrestoException;
+
+import javax.inject.Inject;
+
+import static com.facebook.presto.spi.StandardErrorCode.UNSUPPORTED_ANALYZER_TYPE;
```

Comment:
I prefer not to static import things unless they are very familiar. These are not. They would be clearer in the code without the static import.

Reply:
Presto style is to always static import unless the name is very ambiguous (e.g. we don't static import `of` from `Optional.of` because it's incomprehensible without the prefix, but error code names like this we almost always static import)